src/bitcoin-tx.cpp

```diff
@@ -475,6 +478,8 @@ static void MutateTx(CMutableTransaction& tx, const string& command,
 
     else
         throw runtime_error("unknown command");
+
+    ECC_Stop();
```

Comment:
Let's use a RAII approach here. Otherwise, there will be a leak when control flow leaves the function through an exception.